action/action.go

```diff
@@ -203,6 +202,12 @@ func opFromClaim(action string, stateless bool, c *claim.Claim, ii bundle.Invoca
 	}
 	files["/cnab/app/image-map.json"] = string(imgMap)
 
+	claimBytes, err := json.Marshal(c)
```

Comment:
nit: in the claim store we use `MarshalIndent` [instead](https://github.com/cnabio/cnab-go/blob/master/claim/claimstore.go#L38). Maybe we could factorize the code and add it as a claim function, like we do on [bundles](https://github.com/cnabio/cnab-go/blob/master/bundle/bundle.go#L64) ?

Reply:
Created https://github.com/cnabio/cnab-go/issues/195 to track this item